platform/darwin/src/http_file_source.mm

```diff
@@ -30,52 +25,52 @@ @implementation MBGLBundleCanary
 class HTTPRequestShared {
 public:
     HTTPRequestShared(Response& response_, util::AsyncTask& async_)
-        : response(response_),
-          async(async_) {
+    : response(response_),
+    async(async_) {
     }
-
+    
```

Comment:
This editor is adding a lot of leading spaces :-|

Reply:
I used to get this a lot - there is a setting in Xcode to ensure this doesn't happen. Happy to compare @m-stephen if that would help.